src/starknetCli.ts

```diff
@@ -210,18 +211,23 @@ export async function runStarknetCallOrInvoke(
     }
     throw e;
   }
-
   try {
-    execSync(
+    let warpOutput: string = execSync(
       `${warpVenvPrefix} starknet ${callOrInvoke}  --address ${options.address} --abi ${abiPath} --function ${funcName} --network ${options.network} ${wallet} ${account} ${inputs}`,
-      { stdio: 'inherit' },
-    );
+    ).toString('utf-8');
+
+    if (isCall && !options.use_cairo_abi) {
+      warpOutput = (
+        await decodeOutputs(filePath, options.function, warpOutput.toString().split(' '))
+      ).toString();
+    }
```

Comment:
since decodeOuputs can return objects calling .toString on retuns [object Object]
you'll need a function that converts a Result type to string